arrocke/gloss-translation
# feat: import strongs definitions

## What has changed



Added script to insert strongs definitions into the database. Updated seed dump file.

## Connected Issues



Closes #165

## QA Steps



1. Run `nx run db:prisma migrate reset`
2. In your SQL database, `SELECT * FROM public."LemmaResource" WHERE "resourceCode" = 'STRONGS';`
     - The strongs entries should be shown


## Post-Deployment



- [ ] Add strongs data to the production DB
